config/fallback.go

```diff
@@ -0,0 +1,42 @@
+package config
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/buildwithgrove/path/protocol"
+)
+
+// FallbackURLs is a map of service IDs to fallback URLs.
+// It is unmarshaled from the YAML field `fallback_urls`.
+//
+// Example YAML:
+//
+//	fallback_urls:
+//	  eth: https://eth.rpc.grove.city/v1/1a2b3c4d
+type FallbackURLs map[protocol.ServiceID]string
```

Comment:
I recommend avoiding new types unless they server a very specific purpose: Doesn't seem like Fallback URLs should be a self managing entity.